cats/cats-core/src/main/java/com/netflix/spinnaker/cats/agent/DefaultAgentScheduler.java

```diff
@@ -63,8 +63,15 @@ public DefaultAgentScheduler(ScheduledExecutorService scheduledExecutorService,
 
     @Override
     public void schedule(Agent agent, AgentExecution agentExecution, ExecutionInstrumentation executionInstrumentation) {
+        Long agentInterval = interval;
```

Comment:
Are there other schedulers?

Reply:
There are other schedulers, but they all use the `DefaultAgentIntervalProvider` to acquire the `Interval` to execute the caching agents, e.g. https://github.com/spinnaker/clouddriver/blob/master/cats/cats-redis/src/main/java/com/netflix/spinnaker/cats/redis/cluster/ClusteredAgentScheduler.java#L70

Reply:
Ah nice